low-level/message_handlers/real_stor_encl_msg_handler.py

```diff
@@ -308,6 +313,25 @@ def _generate_logical_volume_alert(self, json_msg, host_name, alert_type, alert_
             self._logical_volume_sensor_message
         self._write_internal_msgQ(EgressProcessor.name(), json_msg, self._event)
 
+    def _generate_disk_group_alert(self, json_msg, host_name, alert_type, alert_id,
+                                                   severity, info, specific_info, sensor_type):
+        """Parses the json message, also validates it and then send it to the
+           egress processor"""
+
+        self._log_debug(f"RealStorEnclMsgHandler, _generate_disk_group_alert,\
```

Comment:
we use `logger.debug` now.

Reply:
why not self._log_debug?
Rest of the functions we are using self._log_debug only.

Reply:
Now I am confused, which one to use. 😂
under `low-level/framework` we have the `Debug` class and it has few methods among which is `_log_debug`, this method is just a wrapper around `logger.debug()` which simply prepends the class name before the message. So are we still using that class? Palak is driving a PR where we are opting cortx logger and we have our  service_logger.Logger wrapper on top of that to reduce the number of changes. So are we going to use this Debug class which is wrapper again on that or should we use directly logger?

Reply:
Okay, If Palak is driving any PR related to this then she will make changes to everyware.